### DISCUSSION TITLE
NFT

### DISCUSSION THREAD

**Discussion by phuldev2** [community/community#20133] [Discussions] [answered]
What is NFT?

**Comment by sthakrishnakumar** [comment, marked as answer]
Non-Fungible Tokens (NFT) are units of data that are stored on a blockchain. Non-fungible means they cannot be replaced by another identical item, which means they are unique.
For example, if an artist wants to sell his/her digital art online, he/she can convert it into NFT and then can sell it. People can buy this artwork using cryptocurrency. Buying digital artworks as NFTs will ensure that they are the official owners. They can sell it for a higher price to another person. So, there will be only one official owner for the NFT at a time.
Any digital work can be converted into NFTs. Music, video clips, images, URLs, tickets and virtual lands in metaverse are a few more examples of the things that are being converted into NFTs.
Ethereum was the first blockchain to support NFTs. That’s why the Ethereum blockchain is mostly used for NFTs. Now many other blockchains are adding support for NFTs due to their increasing popularity.
